### PR TITLE
ENH: Allow bundle aggregation and put load_afq_data output into a named_tuple

### LINKED ISSUE #78: Add a `GroupMeanTransformer`
We often want mean value in each group. Would be great to have a `GroupMeanTransformer` that gives us that easily.

### PR DESCRIPTION
**Pull request by richford**
Resolves #78 

This PR adds a `bundle_agg_func` parameter to `AFQDataFrameMapper`. ~~It's WIP because I haven't yet exposed that functionality as part of `load_afq_data`.~~

### CODE REVIEW

**Comment by richford** [comment]
I'm willing to eat the -0.01% reduction in coverage.

File: afqinsight/datasets.py [hunk 1]
```diff
@@ -81,32 +98,43 @@ def load_afq_data(
...
-        Class labels for each column specified in ``label_encode_cols``.
-        Returned only if ``unsupervised`` is False
+        sessions : list
+            Session IDs. This will be None if ``return_sessions`` is False.
+
+        classes : dict
+            Class labels for each column specified in ``label_encode_cols``.
+            This will be None if ``unsupervised`` is True
+
+        bundle_means : np.ndarray
```
Comment (arokem) [inline afqinsight/datasets.py:135]: Should we put bundle_means into X instead, if `return_bundle_means` is True? And adjust `groups`/`feature_names` accordingly as well?

**Review by arokem — COMMENTED** [review]
Looks good. I had a slightly different API in mind - see comment.

File: afqinsight/datasets.py (hunk 1 above)
Comment (richford) [reply afqinsight/datasets.py]: Good point. Fixed in the latest commit.

**Comment by arokem** [comment]
LGTM! Merging.

**Pull request merged**